docs/v0.5/en/user-guide/concepts/features-that-you-concern.md

```diff
@@ -1,17 +1,5 @@
 # Features that You Concern
 
-## Does GreptimeDB support updates?
```

Comment:
Sorry, I want to know why deleted these questions. They are the questions that the users are concerned.

Reply:
I consider it is more intuitive to move the content to `/user-guide/write-data/overview`, and it would be convenient to have the subdocuments of "write-data" linked to the features.
We can still preserve the questions here and link the answers to `/user-guide/write-data/overview`.

Reply:
@killme2008 updated, PTAL